### DISCUSSION TITLE
[@types/dockerode] missing pruneBuilders

### DISCUSSION THREAD

**Discussion by stanislavHamara** [DefinitelyTyped/DefinitelyTyped#70509] [Issues with a @types package]
`pruneBuilders` is missing in types even though it is present in dockerode https://github.com/apocas/dockerode/blob/db17d1fafe8317bbc41324164cc021f76a43e520/lib/docker.js#L834

[a comment by an automated account is not shown]

**Comment by joebowbeer** [comment]
`pruneBuilder` (singular) is the missing method.

Its type should be something like:

```
pruneBuilder(options: {}, callback: Callback<Dockerode.PruneBuilderInfo>): void;
```

However, I don't know what the `PruneBuilderInfo` type should look like.
